docs/tables/github_code_owner.md

```diff
@@ -0,0 +1,44 @@
+# Table: github_code_owner
+
+All of your code owners rules defined in your repository, in CODEOWNERS files.
+
+The `github_code_owner` table can be used to query information about **ANY** repository, and **you must specify which repository** in the where or join clause (`where repository_full_name=`, `join github_code_owner on repository_full_name=`).
+
+## Examples
+
+### Get All your CodeOwners rules about a specific repository
+
+```sql
+select 
```

Comment:
@aminvielledebatAtBedrock Can you please format this query using https://www.freeformatter.com/sql-formatter.html? There's more information about our doc example standards in https://steampipe.io/docs/develop/table-docs-standards#style-conventions-1.

Reply:
Thanks for the links !